app/views/paye/PayeGovernmentSpendingView.scala.html

```diff
@@ -44,8 +44,8 @@ <h2 class="heading-xlarge flush--top">@Html(Messages("paye.generic.tax_year_head
 
 	<dl class="form-group" id="bar-chart">
 	@for(rowModel <- viewModel.orderedSpendRows){
-		<dt class="percentage" id="@rowModel.category">@Messages("paye.ats.treasury_spending.table." + rowModel.category) (@rowModel.spendData.percentage%)</dt>
-		<dd class="value"><span class="bar-chart" aria-hidden="true" style="width: @{rowModel.spendData.percentage*2}%;"></span> <strong>&pound;@rowModel.spendData.amount.toHalfRoundedUpAmount</strong></dd>
+		<dt class="percentage" id="@rowModel._1">@Messages("paye.ats.treasury_spending.table." + rowModel._1) (@rowModel._2.percentage%)</dt>
```

Comment:
Is there a way to represent this data clearer? 

Perhaps wrapped up in a case class removing the need for `_1` and `_2`

Reply:
yes we can
removed _1 and _2 and wrapped up in a case class:)